src/openms/include/OpenMS/QC/QCBase.h

```diff
@@ -179,19 +179,33 @@ namespace OpenMS
     virtual Status requires() const = 0;
 
     ///function, which iterates through all PeptideIdentifications of a given FeatureMap and applies a given lambda function
-    template <typename T>
-    static void iterateFeatureMap(FeatureMap& fmap, T lambda)
+    template <typename MAP, typename T>
```

Comment:
document that: `PeptideIdentifications without PeptideHits are not passed to the Lambda function. The Lambda may or may not change the PeptideIdentification.`